voyager_archive/templates/voyager_archive/marc_display.html

```diff
@@ -11,18 +11,33 @@
 </form>
 <hr/>
 
-{% comment %}
-Raw Marc dump
-{% endcomment %}
+<div class="row">
+    <div class="column">
+        <h2>Record Leader: {{ marc_record.leader }}</h1>
+        <table class="data-display">
+            <tr>
```

Comment:
I recommend removing the table heading row.  The column headings are not important; users know the 3 parts, and they're in the conventional tag - indicators - field order.  This will tighten up the display, but I think it will still be at least as readable as it is now.

If column headings do prove more readable, then shorten `Indicators` to `Ind` and `Data and Subfields` to `Field`.